### DISCUSSION TITLE
Snapshot URL for Camera streaming software?

### DISCUSSION THREAD

**Discussion by pyrodex** [blakeblackshear/frigate#6564] [Ask A Question]
I use IPCams for iOS/tvOS/macOS to see my cameras all in a single dashboard and/or remotely away from the house. I've got the software using the re-streamed go2rtc stream without issue but my software supports something called a Snapshot URL which shows a quick shot of the camera before pulling it up and using the resources. 

Does go2tc have this notion I can program into my IPcams software so I get a quick snapshot before pulling the camera up fully?

Thanks!

**Comment by NickM-27** [comment]
Go2rtc 1.5 and later support jpeg snapshots for all streams

**Reply by pyrodex** [reply]
> Go2rtc 1.5 and later support jpeg snapshots for all streams

Looks like the container is at 1.2.0, are there future plans to upgrade? I guess I could always run it in a container but like the all in one piece.

**Reply by NickM-27** [reply]
You can of course change the version that runs inside frigate https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

also 0.13 dev is already updated to include 1.5
